benchmarks/dynamo/runner.py

```diff
@@ -1296,13 +1296,17 @@ def comment_on_gh(self, comment):
             f.write(comment)
             filename = f.name
 
+        issue_number = "681"
+        if self.args.dtypes[0] == "float32":
+            issue_number = "2049"
+
```

Comment:
Those are nice numbers. Both factorized to just pair of primes, so can be used as very weak pubkey ;)